### PR TITLE
Russia

### LINKED ISSUE #373: New Entry for Russia
New Entry for Russia

Submitted by [Eugene Altovsky](mailto:pr@ifap.ru):

~~~yaml
---
country:
  en: Russia
  # Manual enter other country names: 
updated: 2019-07-22
updatemsg:
# Related page:
province: 
policies:
  - title:
      en: Requirements on Accessibility for Visually Impaired Persons
    url: http://ifap.ru/library/gost/528722012.pdf
    updated: 2014
    wcagver: WCAG 2.0 derivate, 
    enactdate: 2014
    type: Voluntary standard
    ministries:
      - title:
          ru: "Federal Agency for Technical Regulation and Metrology"
        url:
          ru: "http://www.gost.ru"
    webonly: true
    scope: 
    standard:
      - title:
          en: ""
        desc: ""
        url:
          en: ""
    documents:
      - title:
          en: ""
        desc: ""
        url:
          en: ""
---
~~~

### PR DESCRIPTION
**Pull request by yatil**
Sorry for the somewhat messy Pull Request, will fix #373.

The (main) file that was changed is here: https://github.com/w3c/wai-policies-prototype/pull/379/files#diff-b21ec77cf6ae0e9b71546839eb239b3d

The preview is here: https://deploy-preview-379--wai-policies-prototype.netlify.com/policies/russia/

### CODE REVIEW

**Comment by yatil** [comment]
Deploy preview for *wai-policies-prototype* processing.

Building with commit a3c5d73b82f4ba4224f6f9670e59e271088c2964

https://app.netlify.com/sites/wai-policies-prototype/deploys/5dd7da3d06cd420007669031

**Comment by AndrewArch** [comment]
my error - we should have included the EN name for the policy too
"GOST R 52872-2012 Internet resources. Accessibility requirements for the visually impaired"

**Comment by yatil** [comment]
@AndrewArch I updated the English title accordingly.

**Comment by AndrewArch** [comment]
Thanks Eric - I think we're good to go

**Comment by yatil** [comment]
@slhenry I addressed your comments in https://github.com/w3c/wai-policies-prototype/pull/379/commits/858aae05834c5ad20c11001933c76f4f80edd509 – please let me know when ready to publish.

**Comment by AndrewArch** [comment]
Eric, thanks for modifications from Shawn and me. Good to go now pending Judy's ok

**Comment by yatil** [comment]
@slhenry Is this now OK to publish?

**Comment by slhenry** [comment]
I'm still struggling with pointing to such inaccessible and un-usable content. I'm thinking about putting the PDF link with the first link, and how to word it. Suggestions welcome.

**Comment by slhenry** [comment]
Update: I talked to Judy a few weeks ago and she is even more uncomfortable than I am linking to such inaccessible info. She suggested contacting them and asking them to provide the info in accessible format. I'm not sure that we have direct contact?

We ran out of time before figuring out another approach, and have not been able to get back to it (and won't for a while). Sorry.

**Comment by yatil** [comment]
The disclaimer language Judy came up with is now:

> <p>The laws and/or policies linked below have been submitted by the community, and have not been reviewed for accessibility. In the event that a law or policy may have accessibility problems, W3C/WAI welcomes information on more accessible versions of the linked laws/policies listed via our <a href="https://www.w3.org/WAI/policies/submission/">submission form</a>. We will also retain a link to the official version for each country.</p>

I hope this gets consensus. @AndrewArch, @slhenry?

**Comment by AndrewArch** [comment]
Hi Eric and Shawn,

This sounds like a suitable disclaimer that still allows applicable
laws/polices to be listed.

Thanks.  Andrew

----------------
Andrew Arch
  Woodend, Australia

File: _layouts/policy.html [hunk 1]
```diff
@@ -14,18 +35,20 @@
...
 
   {% include sidenav.html %}
 
+
   <main id="main"{% if page.lang %} lang="{{page.lang}}"{% endif %}>
     <h1 class="post-title">{% include multilang-title.html title=page.country %}</h1>
 
     <div class="post-content">
-      {% if page.states %}<p>See below for information about <a href="#states">{{page.statesdesc.en}}</a>.</p>{% endif %}
+      <p>The laws and/or policies linked below have been submitted by the community, and have not been reviewed for accessibility. In the event that a law or policy may have accessibility problems, W3C/WAI welcomes information on more accessible versions of the linked laws/policies listed via our <a href="https://www.w3.org/WAI/policies/submission/">submission form</a>. We will also retain a link to the official version for each country.</p>
```
Comment (iadawn) [inline _layouts/policy.html:50]: @shawna-slh Would like to discuss this at next opportunity if we can

Comment (iadawn) [reply _layouts/policy.html:50]: Move disclaimer to the footer for all pages

**Comment by iadawn** [comment]
Update to Russia has been included other changes have been superceded

**Pull request closed without merging**